setup.py

```diff
@@ -1,21 +1,17 @@
 #!/usr/bin/env python
 
-import sys
 from setuptools import setup
 
-requires = ['six']
+requires = [
+    'six',
+    'python-dateutil>1.0,!=2.0; python_version<"3"',
+    'python-dateutil>=2.7'
```

Comment:
I believe you also need `; python_version>="3"` on this requirement as well (otherwise you'll get duplicate requirement errors in older versions of pip)

Reply:
Sure